task_01.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/en python
+# -*- coding: utf-8 -*-
+"""Cartesian Products in action"""
+
+
+def get_matches(players):
+    """Matches players for epic tic-tac-toe matches.
+
+    Args:
+        players = a list of names
+
+    Returns:
+        tuple of player matchups.
+
+    Examples:
+        >>> import task_01
+        >>> task_01.get_matches(['Harry', 'Howard', 'Hugh'])
+        [('Harry', 'Howard'), ('Harry', 'Hugh'), ('Howard', 'Hugh')]
+
+    """
+    for person1, position1 in enumerate(players):
+    return enumerate(players)
```

Comment:
The contents of a block (eg, `def` or `for` or `if`) should already be intented. But what you want to do here is not return an enumeration but instead return a list of tuples.